06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/index.js

```diff
@@ -0,0 +1,53 @@
+class InvertedIndex {
+
+  constructor() {
+    this.textArray = [];
+    this.index = {};
+  };
+
+  transformToSingles(file) {

```

Comment:
I believe this PR is just for submission for your front end files and not your `InvertedIndex` class. If so it looks very good, but you can do away with the console.logs.